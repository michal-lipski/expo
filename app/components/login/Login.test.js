import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import {Button} from "react-native-elements";
import ConnectedLogin, {Login} from "./Login";
import configureStore from 'redux-mock-store'

const initialState = {};
const mockStore = configureStore();
let store;

beforeEach(() => {
    store = mockStore(initialState)
});


it('renders the loading screen', async () => {
    const tree = renderer.create(<ConnectedLogin store={store}/>).toJSON();
    expect(tree).toMatchSnapshot();
});


