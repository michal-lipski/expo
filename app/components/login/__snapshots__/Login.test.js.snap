// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the loading screen 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "marginLeft": 15,
        "marginRight": 15,
      },
      undefined,
      undefined,
      undefined,
    ]
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        undefined,
      ]
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#9E9E9E",
            "flexDirection": "row",
            "justifyContent": "center",
            "padding": 19,
          },
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Object {
            "padding": 12,
          },
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 18,
            },
            Array [
              Object {
                "marginRight": 10,
              },
              undefined,
            ],
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        î¡ª
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {},
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            Array [
              Object {
                "color": "white",
                "fontSize": 20,
              },
              undefined,
              Object {
                "fontSize": 17.5,
              },
              undefined,
              undefined,
              undefined,
              undefined,
            ],
          ]
        }
      >
        Login
      </Text>
    </View>
  </View>
</View>
`;
